integrations/haproxy/haproxy_alerts.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: monitoring.googleapis.com/v1
+kind: Rules
+metadata:
+  name: rabbitmq-rules
+  labels:
+    app.kubernetes.io/component: rules
+    app.kubernetes.io/name: rabbitmq-rules
```

Comment:
rabbitmq --> haproxy throughout this file?

Reply:
Of course. Started with rabbit but pivoted to haproxy :facepalm: